<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MESSAGELOGVIEWERWIDGETUI_C</class>
 <widget class="QWidget" name="MESSAGELOGVIEWERWIDGETUI_C">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Message log viewer widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="m_pTableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_pHLayout">
     <item>
      <widget class="QCheckBox" name="m_pDisableAutoRefreshCheckBox">
       <property name="text">
        <string>Disable &amp;autorefresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_pButtonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::NoButton</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MESSAGELOGVIEWERWIDGETUI_C</receiver>
   <slot>ButtonBoxButtonClickedSlot(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ButtonBoxButtonClickedSlot(QAbstractButton*)</slot>
 </slots>
</ui>
